### PR TITLE
Add persistence to consul and couchdb

### PR DESCRIPTION
**Pull request by MichielVanderlee**
Solves Issue #39
This adds 2 requirements prior to install.

The User must define 2 PersistentVolumeClaims in the openwhisk namespace
  - openwhisk-consul
  - openwhisk-couchdb

I've also added an initContainer to consul, to handle the IP change.

### CODE REVIEW

**Comment by DDuTCH** [comment]
I saw issue #23, This was a 'until that's done' solution for consul and a potential permanent solution for couchdb. Which would still require persistence.

File: configure/configure_whisk.yml [hunk 1]
```diff
@@ -16,6 +16,6 @@ spec:
       restartPolicy: Never
       containers:
       - name: configure-openwhisk
-        image: danlavine/whisk_config:v1.5.6
+        image: mastermikeyboy/whisk_config:v1.6.2-dev
```
Comment (bbrowning) [inline configure/configure_whisk.yml]: Is this an intended switch to a different configure image? Or accidentally included in this PR?

Comment (DDuTCH) [inline configure/configure_whisk.yml]: oops, that's accidental.

**Comment by dgrove-oss** [comment]
close as out-of-date?

**Comment by csantanapr** [comment]
>close as out-of-date?

thanks @dgrove-oss for looking into this one.

**Pull request closed without merging**